sdk/javascript/test/utils/arrayHelpers_spec.js

```diff
@@ -0,0 +1,265 @@
+const arrayHelpers = require('../../src/utils/arrayHelpers');
+const { expect } = require('chai');
+
+describe('arrayHelpers', () => {
+	describe('alignUp', () => {
```

Comment:
region around alignUp tests too please

Reply:
describe should form a region itself, no? (if not, you should consider editor change 🤣 )

it seems senseless to create regions around single describe

Reply:
yes, but for symmetry